integration-tests/_tests/12-auth-create-user.test.ftd

```diff
@@ -0,0 +1,85 @@
+;; require fastn built using --features auth flag
+;; FASTN_DB_URL, FASTN_ENABLE_AUTH must be set
+-- import: fastn
+
+-- fastn.test: create user
+
+-- fastn.post: Create user with insufficient data should fail
+url: /-/auth/create-user/
+body: {"username": "siddhant"}
+http-status: 400
```

Comment:
Do we handle 400 status code in `ftd.http()`?

Reply:
we don't at the moment. I can do a quick change in `ftd.js` to handle this if you want me to

Reply:
No, we should return 200 error code. This kind of debate about what is the right error code is bike shedding. Would you return 400 for the case when the username entered does not exist? You monster! Not founds are 404! Some some sort of form field errors are 404 and some 400? And what if someone is not allowed to edit a document, would we return 400 or 401? If username is taken, is it same as "for example, malformed request syntax, invalid request message framing, or deceptive request routing" and both should receive same error code, if one did not care of error code, it would have been excusable, but someone claiming to be error code purist doing so? Not so much.

200 for proper scenarios. 4xx for when something is seriously wrong with the client. Form handling is not something is wrong with client. Everything is perfectly fine with client. User data error is not "client error".

Reply:
Thanks for explaining. This makes sense. I have [changed user errors to be 200 OK](https://github.com/fastn-stack/fastn/pull/1708/commits/62da38dccb170a7058cafe21b5b439ad74f8b013)